packages/datetime/examples/dateRangeInputExample.tsx

```diff
@@ -40,8 +44,26 @@ export class DateRangeInputExample extends BaseExample<IDateRangeInputExampleSta
     private toggleSelectAllOnFocus = handleBooleanChange((selectAllOnFocus) => this.setState({ selectAllOnFocus }));
     private toggleSingleDay = handleBooleanChange((allowSingleDayRange) => this.setState({ allowSingleDayRange }));
 
+    private toggleControlledMode = handleBooleanChange((useControlledMode) => {
+        if (this.state.value === undefined) {
+            // switch to a controlled version of an empty value
+            this.setState({ useControlledMode, value: [null, null] });
+        } else {
+            this.setState({ useControlledMode });
+        }
+    });
+
     protected renderExample() {
-        return <DateRangeInput {...this.state} />;
+        const { value, useControlledMode, ...uncontrolledModeProps } = this.state;
+        return (
+            <div>
+                <DateRangeInput
+                    {...uncontrolledModeProps}
+                    onChange={this.handleChange}
+                    value={useControlledMode ? value : undefined}
+                />
+            </div>
```

Comment:
Originally added this `<div>` because I had a label below the `DateRangeInput` for debugging. Can remove this now.